File: Respostas/Bee_1017/1017.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		int tempoViagem, veloMedia, distancia;
		
		Scanner input = new Scanner(System.in);
		
		tempoViagem = input.nextInt();
		veloMedia = input.nextInt();
		
		distancia = veloMedia * tempoViagem;
		
		System.out.printf("%.3f\n", (0.08333333333333333333333333333333 * distancia));
	}
}

File: Respostas/Bee_1017/1017.c
#include <stdio.h>

int main() {	
	int tempoViagem, veloMedia, distancia;
	
	scanf("%d %d", &tempoViagem, &veloMedia);
	
	distancia = veloMedia * tempoViagem;
	
	printf("%.3f\n", (0.08333333333333333333333333333333 * distancia));
	
	return 0;
}